Meadowlark-js/tests/config/unit/jest.config.js

```diff
@@ -6,14 +6,4 @@ module.exports = {
   ...defaultConfig,
   testMatch: ['<rootDir>/**/*.(spec|test).[jt]s?(x)'],
   testPathIgnorePatterns: ["integration|e2e"],
-  collectCoverageFrom: ['<rootDir>/packages/**/src/**/*.ts', '<rootDir>/backends/**/src/**/*.ts', '<rootDir>/services/**/src/**/*.ts'],
-  coverageThreshold: {
```

Comment:
Should we leave this with the same values, even if they are repeated, so that we can increase each one individually?